tfs/src/main/java/hudson/plugins/tfs/TeamCollectResultsPostBuildAction.java

```diff
@@ -51,6 +52,7 @@ public void setRequestedResults(final List<TeamRequestedResult> requestedResults
         this.requestedResults = requestedResults;
     }
 
+    @SuppressFBWarnings(value = "RV_RETURN_VALUE_IGNORED_BAD_PRACTICE", justification = "No matter the result of mkdirs")

```

Comment:
Why? This is a correct issue. Ideally a Java 7 `Files` API should be used instead